src/components/signup-view/signup-view.jsx

```diff
@@ -44,59 +49,99 @@ export const SignupView = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit} noValidate>
-      <label htmlFor="username">
-        Username:*
-        <input
-          type="text"
-          id="username"
-          value={username}
-          onChange={(e) => setUsername(e.target.value)}
-          required
-          minLength={5}
-          pattern="^[a-zA-Z0-9]*$"
-          onInvalid={(e) => {
-            if (e.target.validity.patternMismatch) {
-              e.target.setCustomValidity(
-                "Username can only use alphanumeric characters."
-              );
-            } else {
-              e.target.setCustomValidity("");
-            }
-          }}
-        />
-      </label>
-      <label htmlFor="email">
-        E-Mail:*
-        <input
-          type="email"
-          id="email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}
-          required
-        />
-      </label>
-      <label htmlFor="birthday">
-        Birthday:
-        <input
-          type="date"
-          id="birthday"
-          value={birthday}
-          onChange={(e) => setBirthday(e.target.value)}
-        />
-      </label>
-      <label htmlFor="password">
-        Password:*
-        <input
-          type="password"
-          id="password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}
-          required
-          minLength={8}
-        />
-      </label>
-      <button type="submit">Submit</button>
-    </form>
+    <div class="p-4 rounded-4 bg-body">
+      <h1 class="text-center">Sign up</h1>
+      <Form onSubmit={handleSubmit} noValidate>
+        <Form.Group className="mb-3" as={Row}>
+          <Form.Label
+            column
+            sm={leftColumnWidth}
+            className="pe-5"
+            htmlFor="username"
+          >
+            Username:*
+          </Form.Label>
+          <Col sm={rightColumnWidth}>
+            <Form.Control
+              type="text"
+              id="username"
+              aria-describedby="usernameText"
+              value={username}
+              onChange={(e) => setUsername(e.target.value)}
+              required
+              minLength={5}
+              pattern="^[a-zA-Z0-9]*$"
+              onInvalid={(e) => {
+                if (e.target.validity.patternMismatch) {
+                  e.target.setCustomValidity(
+                    "Username can only use alphanumeric characters."
+                  );
+                } else {
+                  e.target.setCustomValidity("");
+                }
+              }}
+            />
+          </Col>
+          <Col sm={{ span: rightColumnWidth, offset: leftColumnWidth }}>
+            <Form.Text id="usernameText">
+              Usernames must have at least 5 characters and use only
+              alphanumeric characters.{" "}
```

Comment:
❓What the purpose of this code: `{" "}` here?

Reply:
I think this is something Prettier did when the whitespace was needed. It's not needed anymore so I removed it.